tests/negative/security/tls.go

```diff
@@ -185,6 +247,46 @@ func AppendConfigCustomCertHTTP() types.Test {
 	}
 }
 
+func AppendConfigCABundleCertHTTP() types.Test {
```

Comment:
It might be good to add a comment explaining why this negative test is expected to fail.